src/legacy/core_plugins/kibana/migrations.js

```diff
@@ -78,6 +78,28 @@ export const migrations = {
         delete doc.attributes.savedSearchId;
       }
 
+      // Migrate controls
```

Comment:
@chrisdavies Do you think this is the right approach for changes we need to make to migrations? Do you think we should maybe add it as a separate migration with the version `7.0.0-patch1` or something?

Reply:
I think you're right, @spalger, but we can't use `7.0.0-patch.1` because semver sees it as less than 7.0.0, so it won't get run on anyone's machine who's already run 7.0.0. We'll need to do `7.0.1-patch.1`.

Alternatively, we can update migrations along with this PR so that `7.0.0-migration.1` is seen as bigger than `7.0.0` (e.g. 7.0.0 is implicitly seen as `7.0.0-migration.0`). But that feels pretty hacky.

Reply:
https://github.com/elastic/kibana/issues/31264 I opened that to discuss / track how we want to solve this.

Reply:
Last note. Per Court's comment in the issue mentioned above, we don't need to support people upgrading from beta1 to beta2. So, we can modify the 7.0.0 migrations until release.